ansible/roles/test/tasks/lldp_neighbor.yml

```diff
@@ -1,23 +1,26 @@
 - name: Gather LLDP information from all neighbors by performing a SNMP walk
-  lldp_facts: host={{ hname }} version=v2c community={{ snmp_rocommunity }}
+  lldp_facts:
+    host: "{{ hname }}"
+    version: "v2c"
+    community: "{{ snmp_rocommunity }}"
   connection: local
 
 - name: Print LLDP facts from neighbors
   debug: msg="{{ ansible_lldp_facts }}"
 
-# FIXME: use more strict assertions
```

Comment:
I figured out a fairly straightforward way to test the system description, and just made a new commit. Chassis ID, however, can take a few forms, and will require more advanced logic and parsing.